components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/client/authentication/SampleClientAuthenticator.java

```diff
@@ -74,12 +76,18 @@ public int getPriority() {
     @Override
     public String getName() {
 
-        return "SampleClientAuthenticator";
+        return SAMPLE_CLIENT_AUTHENTICATOR;
     }
 
     @Override
     public boolean isEnabled() {
 
         return this.enabled;
     }
+
+    @Override
+    public List<String> getSupportedClientAuthenticationMethods() {
+
+        return Arrays.asList(SAMPLE_CLIENT_AUTHENTICATOR);
```

Comment:
Lets use a more realistic value here. like client_auth_sample

Reply:
Fixed in ef9331d6b71b68ea222d2747bb6d462e626cc762